src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/SignedInfo.cs

```diff
@@ -62,7 +63,7 @@ public object SyncRoot
         // public properties
         //
 
-        public string Id
+        public string? Id
```

Comment:
CanonicalizationMethod below is another example of a property that should be [AllowNull]